### ISSUE TITLE
Angular class binding no longer working with 3.4.5+

### ISSUE THREAD

**Issue by VoigtK** [tailwindlabs/tailwindcss#14011]
With the release of version 3.4.5, and the merged pull request #13959 the [single CSS class binding](https://angular.dev/guide/templates/class-binding#binding-to-a-single-css-class) in angular is no longer recognised as css class.

For example I have html with

```
<div class="flex flex-col" [class.ml-2]="active">
</div>
```

The css class is not recognised by tailwind and therefore not added to the styles.css

Changing the code to 
```
<div class="flex flex-col ml-2">
</div>
```

it is added to the resulting styles.css again.

**Comment by princemaple** [comment]
Just ran into this issue.

**Comment by NetWin** [comment]
Same here

**Comment by danielehrhardt** [comment]
Same here

**Comment by sebsoftpak** [comment]
Also happening with 3.4.6.

**Comment by elirov** [comment]
Me too.

**Comment by rmarquez-modus** [comment]
Same for me.

**Comment by danielehrhardt** [comment]
Fixed with 3.4.7

**Comment by adamwathan** [comment]
Hey! Should be fixed in v3.4.7 👍
